test/OpenTelemetry.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs

```diff
@@ -37,6 +38,90 @@ public void RuntimeMetricsAreCaptured()
             meterProvider.ForceFlush(MaxTimeToAllowForFlush);
             Assert.True(exportedItems.Count > 1);
             Assert.StartsWith(MetricPrefix, exportedItems[0].Name);
+
+            var gcCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.gc.collections.count");
+            var sumReceived = GetLongSum(gcCountMetric);
+            Assert.True(sumReceived == 0);
+
+#if NETCOREAPP3_1_OR_GREATER
+            var gcAllocationSizeMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.gc.allocations.size");
+            Assert.True(GetLongSum(gcAllocationSizeMetric) > 0);
+#endif
+
+#if NET6_0_OR_GREATER
+            Assert.False(exportedItems.Exists(i => i.Name == "process.runtime.dotnet.gc.committed_memory.size"));
+            Assert.False(exportedItems.Exists(i => i.Name == "process.runtime.dotnet.gc.heap.size"));
+
+            var jitCompiledSizeMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.jit.il_compiled.size");
+            Assert.True(GetLongSum(jitCompiledSizeMetric) > 0);
+
+            var jitMethodsCompiledCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.jit.methods_compiled.count");
+            Assert.True(GetLongSum(jitMethodsCompiledCountMetric) > 0);
+
+            var jitCompilationTimeMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.jit.compilation_time");
+            Assert.True(GetLongSum(jitCompilationTimeMetric) > 0);
+#endif
+
+#if NETCOREAPP3_1_OR_GREATER
+            var lockContentionCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.monitor.lock_contention.count");
+            Assert.True(GetLongSum(lockContentionCountMetric) >= 0);
+
+            var threadCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.thread_pool.threads.count");
+            Assert.True(GetLongSum(threadCountMetric) > 0);
+
+            var completedItemsCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.thread_pool.completed_items.count");
+            Assert.True(GetLongSum(completedItemsCountMetric) > 0);
+
+            var queueLengthMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.thread_pool.queue.length");
+            Assert.True(GetLongSum(queueLengthMetric) == 0);
+
+            var timerCountMetric = exportedItems.First(i => i.Name == "process.runtime.dotnet.timer.count");
+            Assert.True(GetLongSum(timerCountMetric) > 0);
```

Comment:
we can assume that there is at least one timer? Maybe we create our own timers, and validate that atleast that many is reported by the metric itself?

Reply:
Done.